src/EventStore/Broadway/Drivers/Dbal.php

```diff
@@ -24,7 +24,8 @@ public function getDriver()
     {
         $configuration = new Configuration();
 
-        $connectionParams = $this->getStorageConnectionParameters();
+        $databaseConnectionName = $this->config->get('broadway.event-store.connection', 'default');
```

Comment:
This needs to resolve the default connection name here, though it's probably worth encapsulating in the `getStorageConnectionParameters` method entirely for now.